README.md

```diff
@@ -64,103 +64,8 @@ However, if you just want to browse without installing anything, check
 out the [PDF version](https://tldr.sh/assets/tldr-book.pdf).
 
 There are also various other clients provided by the community,
```

Comment:
Since we replaced the whole list by just two sentences, they can quickly be overlooked. Maybe we should put it in a separate section, or at least highlight it a bit:
```suggestion
There are also **various other clients** provided by the community,
```

Reply:
Sure i can do it